security/certificate_types_descriptions/machine-config-operator-certificates.adoc

```diff
@@ -0,0 +1,26 @@
+:_content-type: ASSEMBLY
+[id="cert-types-machine-config-operator-certificates"]
+= Machine Config Operator certificates
+include::_attributes/common-attributes.adoc[]
+:context: cert-types-machine-config-operator-certificates
+
+toc::[]
+
+== Purpose
+
+Machine Config Operator certificates are used to secure connections between the RHCOS nodes and the Machine Config Server.
+
+== Management
+
+These certificates are managed by the system and not the user.
+
+== Expiration
+This CA is valid for 10 years.
+
+The issued serving certificates are valid for 10 years.
+
+CA and client certificates are rotated automatically through the use of controllers.
```

Comment:
There is a signer that generates both client and server certificates?

Reply:
@stlaz any details on this signer ?

Reply:
It is, but it gets deleted after bootstrap. We already opened an [RFE](https://issues.redhat.com/browse/RFE-3287) on that.

Reply:
The internal docs at https://github.com/openshift/api/tree/master/tls/docs/MachineConfig%20Operator%20Certificates do not mention a client cert.

Can you point me to it?

Do we need to document a certificate that gets removed after bootstrap?

Reply:
There are two things that are wrong. 

As you pointed there are no client certificates issued by this signer.

The signer cert (what we call CA in the doc) is not rotated [RFE](https://issues.redhat.com/browse/RFE-3287), as we mentioned the private key gets deleted after bootstrap.

Reply:
Made the requested update